### DISCUSSION TITLE
What should I use as a file manager?

### DISCUSSION THREAD

**Discussion by Drogobo** [i3/i3#5016] [Q&A]
All my go-to file managers like dolphin have a weird dark theme that doesn't work properly. I want a file manager that has a dark mode that works properly in i3. What should I pick (or how do I fix this dark mode thing).

**Comment by linnflux** [comment]
I am using Ranger and it has been splendid. https://github.com/ranger/ranger

**Comment by vincentbernat** [comment]
> All my go-to file managers like dolphin have a weird dark theme that 
> doesn't work properly. I want a file manager that has a dark mode that 
> works properly in i3. What should I pick (or how do I fix this dark mode 
> thing).

Thunar should follow your GTK theme.

**Reply by deanr72** [reply]
I second thunar.

**Comment by Merulox** [comment]
You can change the dolphin theme and qt themes with the "qt5ct" application

**Reply by Drogobo** [reply]
it tells me to set an environment variable. how do I do this?

**Reply by Merulox** [reply]
> it tells me to set an environment variable. how do I do this?

This is what a quick search got me: https://bbs.archlinux.org/viewtopic.php?id=259721
Let me know if that helps

**Reply by Drogobo** [reply]
> > it tells me to set an environment variable. how do I do this?
> 
> This is what a quick search got me: https://bbs.archlinux.org/viewtopic.php?id=259721 Let me know if that helps

It did not work. I have no idea why. Still says I need an environment variable.

**Reply by Merulox** [reply]
> > > it tells me to set an environment variable. how do I do this?
> > 
> > 
> > This is what a quick search got me: [bbs.archlinux.org/viewtopic.php?id=259721](https://bbs.archlinux.org/viewtopic.php?id=259721) Let me know if that helps
> 
> It did not work. I have no idea why. Still says I need an environment variable.

I'm just as lost as you are... All I can do is redirect you to this other thread that's slightly more in-depth: https://unix.stackexchange.com/questions/680483/how-to-add-qt-qpa-platformtheme-qt5ct-environment-variable-in-arch-linux

Remember to reboot or logout once you apply the changes

But if you really can't get qt themes to work, then I'd recommend either Thunar or Nemo.

**Reply by orestisfl** [reply]
> > > it tells me to set an environment variable. how do I do this?
> > 
> > This is what a quick search got me: https://bbs.archlinux.org/viewtopic.php?id=259721 Let me know if that helps
> 
> It did not work. I have no idea why. Still says I need an environment variable.

It's always helpful to include a clear, detailed description of what you did and didn't work

**Reply by Drogobo** [reply]
> > > > it tells me to set an environment variable. how do I do this?
> > > 
> > > 
> > > This is what a quick search got me: [bbs.archlinux.org/viewtopic.php?id=259721](https://bbs.archlinux.org/viewtopic.php?id=259721) Let me know if that helps
> > 
> > 
> > It did not work. I have no idea why. Still says I need an environment variable.
> 
> I'm just as lost as you are... All I can do is redirect you to this other thread that's slightly more in-depth: https://unix.stackexchange.com/questions/680483/how-to-add-qt-qpa-platformtheme-qt5ct-environment-variable-in-arch-linux
> 
> Remember to reboot or logout once you apply the changes
> 
> But if you really can't get qt themes to work, then I'd recommend either Thunar or Nemo.

preach girl this worked
edit: it ducks up kde and makes kde look ugly.
